definition_validator/validator.cc

```diff
@@ -942,17 +938,68 @@ class ValidateWalk {
             return;
         }
 
+        vector<core::MethodRef> missingMethods;
         for (auto proto : abstract) {
             if (proto.data(gs)->owner == sym) {
                 continue;
             }
 
-            auto mem = sym.data(gs)->findConcreteMethodTransitive(gs, proto.data(gs)->name);
-            if (!mem.exists()) {
-                if (auto e = gs.beginError(loc, core::errors::Resolver::BadAbstractMethod)) {
-                    e.setHeader("Missing definition for abstract method `{}`", proto.show(gs));
-                    e.addErrorLine(proto.data(gs)->loc(), "defined here");
-                    e.addAutocorrect(suggestMethodDefinition(gs, loc, proto));
+            auto concreteMethodRef = sym.data(gs)->findConcreteMethodTransitive(gs, proto.data(gs)->name);
+            if (concreteMethodRef.exists()) {
+                continue;
+            }
+
+            missingMethods.emplace_back(proto);
+        }
+
+        if (!missingMethods.empty()) {
```

Comment:
Very minor: it might fit the style of this method more if you convert this to

```cpp
if (missingMethods.empty()) {
    return;
}

// ...
```

Reply:
Fixed in 3d526ec.